views/statistics.tmpl

```diff
@@ -0,0 +1,234 @@
+{{define "statistics"}}
+<!DOCTYPE html>
+<html lang="en">
+    {{ template "html-head" printf "Decred Statistics Page"}}
```

Comment:
Remove the word "Page".  That's self evident.  I see this word was used on the agendas page, but it should not have been.